ophyd/controls/areadetector/detectors.py

```diff
@@ -39,9 +41,138 @@
            'URLDetector',
            ]
 
+def set_and_wait(signal, val):
```

Comment:
@stuwilkins This function encapsulates a pattern you used a couple times. But I'm wondering if there is any reason why wouldn't replace `set_and_wait(signal, val)` with `signal.put(val, wait=True)`. @dchabot says that `wait` is only as reliable as the underlying driver. Can we trust area detector to wait correctly until the value has been set?